threescale_api/resources.py

```diff
@@ -652,6 +656,10 @@ def __init__(self, entity_name='system_name', **kwargs):
 class Application(DefaultResource):
     def __init__(self, entity_name='system_name', **kwargs):
         super().__init__(entity_name=entity_name, **kwargs)
+        self.authobj_factories = {
```

Comment:
we talked about underscore in python and agreed on its disputable purpose, anyway I would prefer this to be prefixed with _ to "hide" it (unless proven otherwise)
Also I see this isn't dict of factories anymore (thank you for this), maybe name could be adjusted (to have it more generic, 'typeless'?)

Reply:
Renamed